frontend/src/components/common/Icon.vue

```diff
@@ -0,0 +1,54 @@
+<template>
```

Comment:
As discussed, we'll remove this Icon file later in the new ticket, let's use the carbon icons.